src/OHD.ts

```diff
@@ -384,6 +384,12 @@ export default class OHD {
       };
     });
   }
+  /**
+   * Add `admin_name` to admins list
+   */
+  public addAdminName(player_name = 'whatAboutBob'): Promise<void> {
```

Comment:
A function like this should not be called without passing a value, especially being Administrative. Setting a default makes the parameter Optional.

Reply:
Fair point. What about further validation that the name is an actual player on the server?

Reply:
That's why I am also checking if there is a response value. the RCONParser and Rules folder handle conversion from response strings to objects

Reply:
I will likely also hook it into the Player object to allow `player.addAdmin()`

Reply:
I see, how you're handling it on the kick/ban funcs, then. I'll take a whack at it but I don't usually do much in JS/TS, tho, cross your fingers and take a deep breath!

Reply:
I'm pushing up some changes to the parser with multi-match functionality. Response type of your call will be `CommandSuccess`, which will be in the definitions folder.
As a side effect, the function must have a `.catch()` or on failure it will throw an uncaught exception.

Reply:
Send is a generic, so you can use 
```ts
this.send<CommandSuccess>(`admin add "${playerName}"`);
```